Userland/Libraries/LibWeb/CSS/StyleValue.h

```diff
@@ -340,6 +347,82 @@ class LengthStyleValue : public StyleValue {
     Length m_length;
 };
 
+class CalculatedStyleValue : public StyleValue {
+public:
+    struct CalcSum;
+    struct CalcSumPartWithOperator;
+    struct CalcProduct;
+    struct CalcProductPartWithOperator;
+
+    // FIXME: These aren't used. They are needed for nested calc()'s
```

Comment:
Such types tend to be called "Tags" in C++, there isn't much to fix here except the name; e.g. `CalcNumberSumTag`.